test/resources/testproject/src/com/example/five.clj

```diff
@@ -57,7 +57,3 @@
   (let [{:keys [foo bar] :or {foo "foo"}} (hash-map)]
     [:bar :foo]
     (count foo)))
-
-;; This was causing both find-local-symbol and find-macros to blow up, for
-;; different reasons
-::str/bar
```

Comment:
no idea what this is but it breaks the project for me (tests don't even start running)